src/styles/Button.module.scss

```diff
@@ -46,20 +53,20 @@
       transition: $hover-transition-time;
     }
 
-    &:active{
-      background-color: $neutral-color-85;
-      border-color: $neutral-color-85;
-      transition: $hover-transition-time;
+    &:active {
+      background-color: $neutral-color-65;
+      border-color: $neutral-color-65;
+      transition: $click-transition-time;
     }
   }
 
   &.square {
     border-radius: 0.5rem;
-    padding: 1rem 1.5rem 1rem 1.5rem;
+    padding: 1rem 1.5rem;
   }
 
   &.round {
-    border-radius: 2.3rem;
-    padding: 1rem 2.25rem 1rem 2.25rem;
+    border-radius: calc(#{$button-height} / 2);
+    padding: 1rem 2.25rem;
   }
```

Comment:
Tutaj też, nie ma co powielać kodu.
Oprócz tego nie wiem skąd wytrzasnąłeś to `2.3rem` ale raz że wartość z czapy, a dwa że wartość z czapy która nie miała wpływu. To co zaporponowałem jeszcze do zmianki żeby odejść od wysokości guzika

To dziwne oznaczenie `#{$zmienna}` działa w taki sposób, że możemy na tej zmiennej operować (w tym przypadku funkcją `calc()`, bez `#` by to nie zadziałało, taka ciekawostka)